src/conf.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=import-error
```

Comment:
What is this change for?

Reply:
It was added by my python linter extension. it's not needed though, so I will take it out.